Makefile

```diff
@@ -10,6 +10,12 @@ SHELL := bash # the shell used internally by Make
 # used inside the included makefiles
 BUILD_SYSTEM_DIR := vendor/nimbus-build-system
 
+# Docker image name
+DOCKER_IMAGE_TAG ?= latest
+DOCKER_IMAGE_NAME ?= statusteam/nim-waku:$(DOCKER_IMAGE_TAG)
```

Comment:
Can we clarify that this is nim-waku v1? Otherwise it might get confusing when we have experimental nim-waku v2 images.

Reply:
That's what tags are for. When you want to switch to v2 you'll just change the tag.

Reply:
Yeah, when the time is there, we can add a make target for v2 and add a different tag.

Reply:
couldn't the tag be 1 then? :) latest to me implies it could be either

anyway not a big deal

Reply:
Feel free to change it, I'll be using one named `deploy-test` for the fleet anyway.